convert/.pre-commit-config.yaml

```diff
@@ -1,27 +1,47 @@
 fail_fast: true
 repos:
-  - repo: https://github.com/pycqa/isort
-    rev: 5.9.2
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+      # TODO: Add this check
+      #     -   id: check-yaml
```

Comment:
Could you clarify this?

Reply:
After uncommenting I got errors, which I don't know how to manage:
```
while parsing a block mapping
  in "convert/chart/templates/deployment.yaml", line 31, column 9
did not find expected key
  in "convert/chart/templates/deployment.yaml", line 31, column 44
while parsing a block mapping
  in "convert/chart/templates/mapping.yaml", line 7, column 3
did not find expected key
  in "convert/chart/templates/mapping.yaml", line 7, column 33
while constructing a mapping
  in "convert/chart/templates/service.yaml", line 7, column 14
found unhashable key
  in "convert/chart/templates/service.yaml", line 7, column 15
```

Reply:
Check this thread on stackoverflow: https://stackoverflow.com/questions/54479397/error-converting-yaml-to-json-did-not-find-expected-key-kubernetes

Reply:
Done

Reply:
Done